guides/common/modules/proc_enabling-openscap.adoc

```diff
@@ -11,7 +11,6 @@ To use the OpenSCAP plug-in and content on external {SmartProxies}, you must ena
 ----
 # {installer-scenario-smartproxy} \
 --enable-foreman-proxy-plugin-openscap \
---foreman-proxy-plugin-openscap-ansible-module true \
```

Comment:
I still cannot reproduce this. Tested on Smart Proxy 3.5 with Katello; see https://github.com/theforeman/foreman-documentation/blob/3.5/guides/common/attributes-katello.adoc#L8

````
# foreman-installer --scenario foreman-proxy-content --enable-foreman-proxy-plugin-openscap --foreman-proxy-plugin-openscap-ansible-module true --foreman-proxy-plugin-openscap-puppet-module true
2023-09-29 08:50:41 [NOTICE] [root] Loading installer configuration. This will take some time.
2023-09-29 08:50:44 [NOTICE] [root] Running installer with log based terminal output at level NOTICE.
2023-09-29 08:50:44 [NOTICE] [root] Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG. See --full-help for definitions.
ERROR: Unrecognised option '--foreman-proxy-plugin-openscap-ansible-module'

See: 'foreman-installer --help'
````